### PR TITLE
Added SeenThis page block and custom tag

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | master, 4.6
| Edition       | all

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/block_reference.md [hunk 1]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
 
 On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
 
+## SeenThis! block
+
+SeenThis! block provides an adaptive streaming technology with no limitations as conventional streaming service. It allows to preserve the best video quality with minimum amount of data transfer.
```
Comment (adamwojs) [inline docs/content_management/block_reference.md:229]:
```suggestion
Integration with (SeenThis!)[https://seenthis.co/] service. SeenThis! block provides an adaptive streaming technology with no limitations as conventional streaming service. It allows to preserve the best video quality with minimum amount of data transfer.
```

File: docs/content_management/block_reference.md [hunk 2]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
...
+
+!!! note
+    This page block is in an opt-in bundle, to use it, install the bundle first.
```
Comment (adamwojs) [inline docs/content_management/block_reference.md:232]:
```suggestion
    This page block is in an opt-in bundle, to use it, install `ibexa/connector-seenthis` bundle first.
```

File: docs/content_management/create_edit_content_items.md [hunk 3]
```diff
@@ -115,6 +115,15 @@ add or delete rows and columns in any part of the table, as well as merge and sp
 Your installation can also have custom elements available in the Rich Text field.
 Contact your website administrator about the details of using them.
 
+### SeenThis! streaming
+
+!!! note
+    This custom tag is in an opt-in bundle, to use it, install the bundle first.
```
Comment (adamwojs) [inline docs/content_management/create_edit_content_items.md:121]:
```suggestion
    This custom tag is in an opt-in bundle, to use it, install `ibexa/connector-seenthis` bundle first.
```

File: docs/content_management/block_reference.md [hunk 4]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
...
+- **Big play button** – Set to display large play button.
+- **Autoplay** – Configure whether the video starts automatically.
+- **Muted** – Configure whether the video starts muted.
+- **Play button** – Set the number of products to be displayed.
+- **Mute button** – Set whether the mute button is displayed or not.
+- **Loop** – Set to play a video in a loop mode.
+- **Loop count** – Set the number of loop repetitions.
+- **Include audio** – Set to include or not audio with the video.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:248]:
```suggestion
On the **Design** tab, in the **View** field, select the layout to be used to present a list of products, and submit your changes.
```

File: docs/content_management/block_reference.md [hunk 5]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
...
+
+
+!!! note
+
+    SeenThis technology might be blocked by some ad blocker solutions. If you can't see the block once configured, check ad blocker configuration.
```
Comment (dabrt) [inline docs/content_management/block_reference.md:253]:
```suggestion
    Some ad blocker solutions block content that comes from SeenThis. If you can't see the block after it is configured, check the ad blocker configuration.
```

**Review by dabrt — CHANGES_REQUESTED** [review]

File: docs/content_management/create_edit_content_items.md [hunk 6]
```diff
@@ -115,6 +115,15 @@ add or delete rows and columns in any part of the table, as well as merge and sp
 Your installation can also have custom elements available in the Rich Text field.
 Contact your website administrator about the details of using them.
 
+### SeenThis! streaming
+
+!!! note
+    This custom tag is in an opt-in bundle, to use it, install `ibexa/connector-seenthis` bundle first.
```
[the remark this answers is not in the document]
Comment (juskora) [reply docs/content_management/create_edit_content_items.md:121]: It's a bundle.

File: docs/content_management/block_reference.md [hunk 7]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
 
 On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
 
+## SeenThis! block
+
+Integration with (SeenThis!)[https://seenthis.co/] service. SeenThis! block provides an adaptive streaming technology with no limitations as conventional streaming service. It allows to preserve the best video quality with minimum amount of data transfer.
```
[the remark this answers is not in the document]
Comment (MagdalenaZuba) [reply docs/content_management/block_reference.md:229]: We discussed it as a team last week, we even have a ticket for it https://issues.ibexa.co/browse/IBX-6762, but it is not in Justyna's scope so let's not dwell on that 😃

File: docs/content_management/block_reference.md (hunk 5 above)
Comment (MagdalenaZuba) [inline docs/content_management/block_reference.md:253]:
```suggestion
    SeenThis! technology might be blocked by some ad blocker solutions. If you can't see the block once configured, check the ad blocker configuration.
```

File: docs/content_management/block_reference.md [hunk 8]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
...
+- **Video ID** – Enter the ID of the video to be streamed.
+- **Tracker ID** – Enter the tracking code.
+- **Big play button** – Set to display large play button.
+- **Autoplay** – Configure whether the video starts automatically.
+- **Muted** – Configure whether the video starts muted.
+- **Play button** – Set the number of products to be displayed.
+- **Mute button** – Set whether the mute button is displayed or not.
+- **Loop** – Set to play a video in a loop mode.
+- **Loop count** – Set the number of loop repetitions.
+- **Include audio** – Set to include or not audio with the video.
```
Comment (MagdalenaZuba) [inline docs/content_management/block_reference.md:246]:
```suggestion
- **Include audio** – Configure whether to include audio with the video.
```

File: docs/content_management/block_reference.md [hunk 9]
```diff
@@ -223,6 +224,34 @@ On the **Basic** tab, set values in the following fields:
...
+
+- **Name** – Enter a name for the page block.
+- **Video ID** – Enter the ID of the video to be streamed.
+- **Tracker ID** – Enter the tracking code.
+- **Big play button** – Set to display large play button.
+- **Autoplay** – Configure whether the video starts automatically.
+- **Muted** – Configure whether the video starts muted.
+- **Play button** – Set the number of products to be displayed.
+- **Mute button** – Set whether the mute button is displayed or not.
+- **Loop** – Set to play a video in a loop mode.
```
Comment (MagdalenaZuba) [inline docs/content_management/block_reference.md:244]:
```suggestion
- **Loop** – Set whether to play a video in a loop mode.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Review by julitafalcondusza — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**